app/src/components/tables/ValidatorTable/ValidatorTable.tsx

```diff
@@ -61,19 +62,42 @@ export const ValidatorTable: React.FC<ValidatorTableProps> = ({
     );
   }, [validatorsTableOptions, totalEraValidators]);
 
-  const columns = useMemo<ColumnDef<ValidatorWeight>[]>(
+  const columns = useMemo<ColumnDef<ApiData.ValidatorsInfo>[]>(
     () => [
+      {
+        header: `${t('rank')}`,
+        accessorKey: 'rank',
+        enableSorting: false,
+      },
       {
         header: `${t('public-key')}`,
-        accessorKey: 'public_key',
+        accessorKey: 'publicKey',
+        cell: ({ getValue }) => truncateHash(getValue<string>()),
+      },
+      {
+        header: `${t('fee-percentage')}`,
+        accessorKey: 'feePercentage',
+        enableSorting: false,
+      },
+      {
+        header: `${t('delegators')}`,
+        accessorKey: 'delegatorsCount',
+        enableSorting: false,
+      },
+      {
+        header: `${t('total-stake')}`,
+        accessorKey: 'totalStakeCspr',
+        enableSorting: false,
+      },
```

Comment:
can we have this value formatted so `613804075` will be `613,804,075 CSPR` ? 
And for `CSPR` - I think we should have it as env variable too - as Chengdu for example wanted to disable this at all.

Reply:
Yes! I made a ticket for all the styles/polish that we need for the table (there's much more to format). https://github.com/casper-network/casper-blockexplorer-frontend/issues/211. This was just the minimum to display the data since it was more focused on the middleware work.